### ISSUE TITLE
Reminders crashes on Manjaro

### ISSUE THREAD

**Issue by tijnzor** [cross-platform/icloud-for-linux#52]
Opening reminders via Icloud.com works fine.

**Error Message:**
_APP NAME
reminders2

HOSTING APP NAME
cloudos2

CLIENT TIME
Tue Aug 16 2022 10:34:21 GMT+0200 (Midden-Europese zomertijd) (1660638861039)

USER AGENT
Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) QtWebEngine/5.12.8 Chrome/69.0.3497.128 Safari/537.36

HOSTNAME
www.icloud.com

BUILD INFO
2220Project30__2220B20__nl-nl

SESSION ID
{redacted}

ENVIRONMENT INFO
PROD

REPORTED ERROR TITLE
SyntaxError: Failed to execute 'measure' on 'Performance': The mark 'null' does not exist.

REPORTED ERROR TYPE
UNHANDLED_REJECTION

LINE NUMBER
11

USER APPLE ID
{Redacted}

USER DSID
{Redacted}

STACK
Error: Failed to execute 'measure' on 'Performance': The mark 'null' does not exist.
    at Object.<anonymous> (https://www.icloud.com/applications/reminders2/2220Project30/nl-nl/main.js:11:1655883)
    at https://www.icloud.com/applications/reminders2/2220Project30/nl-nl/main.js:11:1654542
    at Object.next (https://www.icloud.com/applications/reminders2/2220Project30/nl-nl/main.js:11:1654647)
    at a (https://www.icloud.com/applications/reminders2/2220Project30/nl-nl/main.js:11:1653360)

ERROR CONTEXT
undefined_

**Comment by AgustinNormand** [comment]
Same issue here

```
APP NAME
reminders2

HOSTING APP NAME
cloudos2

CLIENT TIME
Wed Aug 17 2022 22:44:04 GMT-0300 (Argentina Standard Time) (1660787044242)

USER AGENT
Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) QtWebEngine/5.12.8 Chrome/69.0.3497.128 Safari/537.36

HOSTNAME
www.icloud.com

BUILD INFO
2220Project30__2220B20__es-es

SESSION ID
ca21d664-b7d3-4edb-9452-5915062d811a

ENVIRONMENT INFO
PROD

REPORTED ERROR TITLE
SyntaxError: Failed to execute 'measure' on 'Performance': The mark 'null' does not exist.

REPORTED ERROR TYPE
UNHANDLED_REJECTION

LINE NUMBER
11

USER APPLE ID
normandagustin@icloud.com

USER DSID
8405166828

STACK
Error: Failed to execute 'measure' on 'Performance': The mark 'null' does not exist.
    at Object.<anonymous> (https://www.icloud.com/applications/reminders2/2220Project30/es-es/main.js:11:1655876)
    at https://www.icloud.com/applications/reminders2/2220Project30/es-es/main.js:11:1654535
    at Object.next (https://www.icloud.com/applications/reminders2/2220Project30/es-es/main.js:11:1654640)
    at a (https://www.icloud.com/applications/reminders2/2220Project30/es-es/main.js:11:1653353)

ERROR CONTEXT
undefined
```

**Comment by MarcusTomlinson** [comment]
This should be fixed in v0.12 now. Please run `snap refresh icloud-for-linux`.
